boards/mips/qemu_mipsel/qemu_mipsel.yml

```diff
@@ -0,0 +1,7 @@
+identifier: qemu_mipsel
```

Comment:
rename this file to be .yaml, we will support yml in the scripts, but right now .yml is being ignored. Once you do that, you will have lots of sanitycheck errors, so we need to fix those.

Reply:
OK fixing that now. I think my toolchain must be picking up mips/cpu from its install dir rather than ext/hal/mips/include/

Last time I checked your MIPS toolchain only supported big-endian targets, so will not compile for the currently written boards/emulators.

Reply:
I fixed the sanity check by adding to the enum in the sanity check script. I have a feeling this is not an acceptable solution, so I will probably need to read up on what those yaml files actually do...

Reply:
yes, out mips toolchain is not well tested, need to fix this. re the enum, I am thinking now this should not be an enum and should be free form instead to allow anyone to add board/toolchain combos without having to change the scripts.